test/wait-for-services.sh

```diff
@@ -14,10 +14,11 @@ done
 if [ -v SELENIUM_REQUIRED ]; 
 then
   echo "Waiting for Selenium..."
-  until $(curl --output /dev/null --silent --head --fail http://selenium:4444/wd/hub); do
+  until $(curl --output /dev/null --silent --head --fail http://selenium:4444/status); do
```

Comment:
I am under the impression that the standard endpoint to use for the health check is `/wd/hub/status` according to the docs.
https://github.com/SeleniumHQ/docker-selenium?tab=readme-ov-file#waiting-for-the-grid-to-be-ready